lddtree.sh

```diff
@@ -25,6 +25,7 @@ usage() {
 	  -R <root>       Use this ROOT filesystem tree
 	  --no-auto-root  Do not automatically prefix input ELFs with ROOT
 	  --no-recursive  Do not recursivly parse dependencies
+	  --no-header     Do not show header (binary and interpreter info)
```

Comment:
readelf or scanelf do not resolve path, it will output only linked lib filename 
lddtree resolve path, as you know, and take care of rpath values